test/metabase/api/revision_test.clj

```diff
@@ -50,9 +50,9 @@
 
 ;; case with no revisions (maintains backwards compatibility with old installs before revisions)
 (deftest no-revisions-test
-  (testing "Loading revisions, where there are no revisions, should work"
+  (testing "Loading revisions, where there are no changes, should work"
```

Comment:
I think the first description was correct. There are no revisions for this card.
```suggestion
  (testing "Loading revisions, where there are no revisions, should work"
```

Reply:
updated in 254f33bedd